release-tag.sh

```diff
@@ -0,0 +1,7 @@
+version=`python setup.py --version`
+eval "$(ssh-agent -s)" #start the ssh agent
+chmod 400 ./openfisca_bot
```

Comment:
I don't think travis let you add an ssh key to the project settings like Circle CI 😢 .

However, encrypted files are [not available for pull requests coming from forks](https://docs.travis-ci.com/user/pull-requests#Security-Restrictions-when-testing-Pull-Requests), so I don't think the kind of attack you describe is possible.

There is still the risk of a mistake from a trusted developper though. Do you think this risk is enough to spend some time migrating to Circle-CI ?